### PR TITLE
fix(docs): Immich quota claim note

### PR DESCRIPTION
**Pull request by aviv926**
Added following https://github.com/immich-app/immich/issues/8073

### CODE REVIEW

File: docs/docs/administration/oauth.md [hunk 1]
```diff
@@ -59,21 +59,21 @@ Before enabling OAuth in Immich, a new client application needs to be configured
...
+| Client Secret                                        | string  | (required)           | Required. Client Secret (previous step)                                                                                   |
+| Scope                                                | string  | openid email profile | Full list of scopes to send with the request (space delimited)                                                            |
+| Signing Algorithm                                    | string  | RS256                | The algorithm used to sign the id token (examples: RS256, HS256)                                                          |
+| Storage Label Claim                                  | string  | preferred_username   | Claim mapping for the user's storage label                                                                                |
+| Storage Quota Claim                                  | string  | immich_quota         | Claim mapping for the user's storage<br />**Note:** Claim is only used on user creation and cannot be changed afterwards. |
```
Comment (jrasm91) [inline docs/docs/administration/oauth.md:71]:
```suggestion
| Storage Quota Claim                                  | string  | immich_quota         | Claim mapping for the user's storage<br />**Note:** Claim is only used on user creation and not synchronized after that. |
```

**Comment by aviv926** [comment]
I still don't use this option personally, but is this note also valid for other things?
Like for example Storage Label Claim?

**Review by jrasm91 — APPROVED** [review]

**Comment by jrasm91** [comment]
> I still don't use this option personally, but is this note also valid for other things? Like for example Storage Label Claim?

Yes, all these claims are set when the user is created and not synchronized after that. Maybe we could add a [1] and then move the note below the table instead.

**Comment by aviv926** [comment]
> > I still don't use this option personally, but is this note also valid for other things? Like for example Storage Label Claim?
> 
> Yes, all these claims are set when the user is created and not synchronized after that. Maybe we could add a [1] and then move the note below the table instead.

 think it would be better, I changed accordingly.

**Pull request merged**